### ISSUE TITLE
Use a permission class to enforce user access for financial aid API

### ISSUE THREAD

**Issue by noisecapella** [mitodl/micromasters#1064]
From #1053

**Comment by pdpinch** [comment]
Is this a refactoring, or does it address a security issue?

Is this the relevant part of the thread: https://github.com/mitodl/micromasters/pull/1053#discussion_r79214148 ?

**Comment by noisecapella** [comment]
Yes that's the right comment. It's a refactoring issue. The code enforces this in the serializer code so there's no immediate security issue and the case is tested for.
